mob.css

```diff
@@ -0,0 +1,466 @@
+}
+
+.bands {
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  color: #fff;
+  height: 120px;
+  background: transparent;
+  width: 100%;
+  padding-top: 23px;
+  margin: 7.5px;
+}
+
+/* Set transparent background to the cards */
+.bands::before {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(19, 19, 19, 0.5);
+}
+
+.bands .icon {
+  font-size: 40px;
+  margin: 23px;
+  z-index: 1;
+}
+
+.bands h3 {
+  color: #ec5242;
+  font-family: Cocogoose, sans-serif;
+  font-weight: lighter;
+  min-width: 90px;
+  width: 155px;
+  margin-right: 10px;
+  z-index: 1;
+}
+
+.bands p {
+  min-width: 211px;
+  margin-left: 13px;
+  margin-right: 13px;
+  z-index: 1;
+}
+
+/* botton inside the main section  */
+.mainButton {
+  text-decoration: none;
+  color: #fff;
+  background: #ec5242;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  width: 310px;
+  height: 80px;
+  line-height: 80px;
+  text-align: center;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+/* Information section */
+.information {
+  margin: 15px;
+  min-width: 530px;
+}
+
+.information .divTitle {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.divTitle h2 {
+  color: #272a31;
+  margin-top: 25px;
+}
+
+.divTitle div {
+  margin-bottom: 10px;
+}
+
+/* Div that contains the information - Image and details */
+.divCard {
+  display: flex;
+  justify-content: flex-start;
+  width: 517px;
+  margin-top: 40px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.divImg {
+  min-width: 175px;
+  height: 170px;
+  flex-shrink: 0;
+  background: url("../images/background/chess.png") no-repeat;
+  background-size: 80px;
+  position: relative;
+}
+
+.contImg {
+  width: 145px;
+  height: 145px;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.contImg img {
+  width: 100%;
+  height: 100%;
+}
+
+.divInfo {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-left: 25px;
+  margin-right: 35px;
+  color: #272a31;
+}
+
+/* Style the content inside the divInfo container */
+.divInfo h3 {
+  font-weight: 900;
+  font-size: 18px;
+}
+
+.divInfo h4 {
+  color: #ec5242;
+  margin-top: 5px;
+}
+
+.separate {
+  width: 20px;
+  height: 2px;
+  background: #777;
+  margin-top: 13px;
+}
+
+.divInfo p {
+  font-weight: 600;
+  margin-top: 18px;
+}
+
+/* Footer section */
+footer {
+  background: #414246;
+  display: flex;
+  flex-direction: column;
+  min-width: 530px;
+}
+
+.sponsorCont {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-bottom: 15px;
+}
+
+.sponsorDiv {
+  width: 140px;
+  flex-shrink: 0;
+  height: 60px;
+  margin: 15px;
+}
+
+.sponsorImg {
+  width: 100%;
+  display: block;
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.last {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #eee;
+}
```

Comment:
Great design @sonuengineer 🤝

- [ ] Kindly make sure the footer section of your about for the desktop view replicates the design given. You may change the footer color here to a dark color.
- Your design
![Your-about-footer](https://user-images.githubusercontent.com/63917725/205430628-dbdc4a5e-824e-404d-b2b8-bcefa7d37153.PNG)
- Given design
![Given-about-footer](https://user-images.githubusercontent.com/63917725/205430637-b027f93a-ffc6-4256-add3-a39c635d4bc1.PNG)